3-terrarium/2-intro-to-css/README.md

```diff
@@ -94,7 +96,7 @@ h1 {
 
 This way of styling a tag gives you control over unique elements, but you need to control the styles of many plants in your terrarium. To do that, you need to leverage CSS selectors.
 
-### Ids
+### Id's
 
```

Comment:
This is not a possessive word, but rather a plural, so we don't need the apostrophe